crates/bevy_ui/src/layout/mod.rs

```diff
@@ -1123,7 +1088,7 @@ mod tests {
             .collect::<Vec<Entity>>();
 
         for r in [2, 3, 5, 7, 11, 13, 17, 19, 21, 23, 29, 31].map(|n| (n as f32).recip()) {
-            let mut s = r;
+            let mut s = 1. - r;
```

Comment:
```suggestion
            // This fails with very small / unrealistic scale values
            let mut s = 0.2 + r;
```

Tiny suggestion for the test.